libpermian/plugins/compose/__init__.py

```diff
@@ -12,7 +12,8 @@
 from ..beaker import list_tagged_composes
 from libpermian.events.structures.builtin import ProductStructure
 from libpermian.events.structures.base import BaseStructure
-
+from libpermian.plugins.kickstart_test import BootIsoStructure
```

Comment:
I know this is not subject of this pull request, but the BootIsoStructure should not be part of kickstart_test but rather it's own plugin so that it's not dependent on a workflow.